### PR TITLE
support aarch64

### PR DESCRIPTION
**Pull request by gao-feng**
Signed-off-by: Gao feng <omarapazanadi@gmail.com>

### CODE REVIEW

**Comment by laijs** [comment]
failed on gofmt

**Comment by gnawux** [comment]
could this be cross compiled? or we need to build it on an arm64 box?

**Comment by gao-feng** [comment]
@gnawux 

Had better build on arm64 box. It's tested.

File: hypervisor/qemu/qemu_arm64.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	if boot.HotAddCpuMem {
+		memParams = fmt.Sprintf("size=%d,slots=1,maxmem=%dM", boot.Memory, hypervisor.DefaultMaxMem)
+		cpuParams = fmt.Sprintf("cpus=%d,maxcpus=%d", boot.CPU, hypervisor.DefaultMaxCpus)
+	} else {
+		memParams = strconv.Itoa(boot.Memory)
+		cpuParams = strconv.Itoa(boot.CPU)
+	}
+
+	return []string{
+		"-machine", "virt,usb=off", "-cpu", "cortex-a57",
```
Comment (gnawux) [inline hypervisor/qemu/qemu_arm64.go]: for those have `/dev/kvm`, this line should be 

```		"-machine", "virt,accel=kvm,usb=off", "-cpu", "host",```

For example, my debian on pine64

**Comment by gnawux** [comment]
Looks good for my debian pine64.

- self built official qemu 2.6.2 
- go 1.6.2
- kernel 3.10
- storage driver: devicemapper

With kvm enabled (no vm cache):

```
root@pine64:/home/debian# time hyperctl run -t aarch64/busybox sh -c "echo hello, arm64"
hello, arm64

real	0m4.381s
user	0m0.130s
sys	0m0.080s```

**Comment by gao-feng** [comment]
@gnawux updated,  thanks!

**Comment by gnawux** [comment]
I think this could be merged when https://github.com/hyperhq/hyperstart/pull/207 is merged.

And we should update the Godep of hyperhq/hyperd

**Pull request merged**